src/go/rpk/pkg/cli/cmd/cluster/maintenance/disable.go

```diff
@@ -55,7 +57,25 @@ func newDisableCommand(fs afero.Fs) *cobra.Command {
 			}
 
 			out.MaybeDie(err, "error disabling maintenance mode: %v", err)
-			fmt.Printf("Successfully disabled maintenance mode for node %d\n", nodeID)
+
+			retries := 3
+			for {
+				b, err := client.Broker(cmd.Context(), nodeID)
+				if err != nil {
+					if retries <= 0 {
+						out.Die("Exiting.. error retrieving broker status after emitting the request: %v", err)
```

Comment:
```suggestion
						out.Die("Error retrieving broker status after emitting the request: %v", err)
```
wdyt?

Reply:
Indeed. Since we die, no need to say `Exiting` explicitly.